docs/data-ai/quickstart/set-up-viam.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
To also save space in the left nav and limit duplication, I'd move this to a "requirements" section and then link to our existing pages that cover setup.